travis/get_profile_stats.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import pstats
+import sys
+
+
+def print_stats(filter_fnames=None, exclude_fnames=None,
+                sort_index=0, limit=0):
+    """Print stats with a filter or exclude filenames, sort index and limit
+    :param filter_fnames: List of relative paths to filter and show them.
+    :param exclude_fnames: List of relative paths to avoid show them.
+    :param sort_index: Integer with `pstats tuple` index to sort the result.
+    :param limit: Integer to limit max result.
+    :return: Directly print of `pstats` summarize info.
```

Comment:
Why not return a message list instead of directly printing?
That would be reuse friendly.